cmd/docker-app/root.go

```diff
@@ -5,35 +5,14 @@ import (
 	"strings"
 
 	"github.com/docker/app/internal"
+	"github.com/docker/cli/cli/command"
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 )
 
-var (
-	commands = []*cobra.Command{
-		deployCmd(),
-		helmCmd(),
-		initCmd(),
-		inspectCmd(),
-		pushCmd(),
-		renderCmd(),
-		saveCmd(),
-		versionCmd(),
-	}
-	experimentalCommands = []*cobra.Command{
-		imageAddCmd(),
-		imageLoadCmd(),
-		loadCmd(),
-		mergeCmd(),
-		packCmd(),
-		pullCmd(),
-		splitCmd(),
-		unpackCmd(),
-	}
-)
-
 // rootCmd represents the base command when called without any subcommands
-func newRootCmd() *cobra.Command {
+// FIXME(vdemeester) use command.Cli interface
```

Comment:
Is it the Initialize in the Deploy command which prevent you to use command.Cli interface?

Reply:
@silvin-lubecki yes :sweat_smile: I'm preparing to bump the `docker/cli` and upstream a lot of changes, that's why I put the FIXME :angel: a way to say "it's gonna go at some point :wink:"